django_rq/views.py

```diff
@@ -58,6 +60,12 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
+def _get_all_workers(config, workers_collections):
+    for collection in workers_collections:
```

Comment:
I took the liberty to put the function `_get_all_workers` in `django_rq/workers.py` as `get_all_workers_by_configuration()`. I think it got more expressive.